src/rust/plugin-registry/src/static_files/plugin.nomad

```diff
@@ -150,27 +164,16 @@ job "grapl-plugin" {
 
       artifact {
         source      = var.rootfs_artifact_url
-        destination = "local/"
-        headers {
-          x-amz-expected-bucket-owner = var.aws_account_id
-          x-amz-meta-client-id        = "nomad-deployer"
-        }
-      }
-
-      artifact {
-        source      = var.plugin_artifact_url
-        destination = "local/plugin"
-        mode        = "file"
+        destination = "./"
         headers {
           x-amz-expected-bucket-owner = var.aws_account_id
           x-amz-meta-client-id        = "nomad-deployer"
         }
       }
 
       config {
-        KernelImage = "local/vmlinux"
-        BootDisk    = "local/rootfs.ext4"
-        //        Disks = [ "local/plugin" ]
+        # KernelImage and BootDisk are the default values of vmlinux/rootfs.ext4
+        # (unfortunately, I kept having trouble setting them explicitly)
```

Comment:
basically the loop for changing this file is like. 15 minutes. if I set KernelImage = "vmlinux" it would work but if i set BootDisk = "rootfs.ext4" it would fail. just bonkers stuff all around

Reply:
😭